app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -58,6 +58,17 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler {
         setupNav()
 
         handleNotifications(intent)
+
+        if (settingsHandler.showPasscodeBanner) {
+            setupPasscode()
+            settingsHandler.showPasscodeBanner = false
```

Comment:
this user won't see showPasscodeBanner anymore and this functionality can be removed?

Reply:
I use a separate boolean.